alluxio/worker_ring.py

```diff
@@ -111,7 +98,7 @@ def dump_main_info(self):
 
 
 class EtcdClient:
-    PREFIX = "/DHT/DefaultAlluxioCluster/AUTHORIZED/"
+    PREFIX = "/DHT/DefaultAlluxioCluster/"
```

Comment:
this is the line to change from all worker to active worker list? could not find other logics related in the PR

Reply:
yes, it would return as the following KV
```
/ServiceDiscovery/DefaultAlluxioCluster/worker-cb157baa-afe0-4b98-8af0-1a4645d38456
{"Identity":{"version":1,"identifier":"cb157baaafe04b988af01a4645d38456"},"WorkerNetAddress":{"Host":"192.168.4.36","ContainerHost":"","RpcPort":29999,"DataPort":29997,"SecureRpcPort":0,"NettyDataPort":29997,"WebPort":30000,"DomainSocketPath":""},"State":"AUTHORIZED","GenerationNumber":-1,"ServiceEntityName":"worker-cb157baa-afe0-4b98-8af0-1a4645d38456"}
```